README.md

```diff
@@ -59,16 +63,24 @@ This repository contains helper scripts and commands to create meetup videos for
 
         $ ffmpeg -i joined.mp4 -i final.wav -shortest -preset ultrafast -q 0 final.mp4
 
-* scale video to 16:9 
+* Managing volume in the video
+
+        $ ffmpeg -i <input_video_name> -af 'volume=2.5<or 1x>' -preset ultrafast <output_video_name>
+
+* scale video to 16:9
 
         $ ffmpeg -i final.mp4 -vf pad="ih*16/9:ih:(ow-iw)/2:(oh-ih)/2" -preset ultrafast -acodec copy final2.mp4
 
-* Generate the title image 
+    or specify absolute pixels
+
+        $ ffmpeg -i <input_video_name> -vf "scale=1024:768" -preset ultrafast <output_video_name>
```

Comment:
why the scaling ? this is not the native aspect ration to be uploaded too. 16:9 is the preferred resolution for the final video.